lib/galaxy/schema/workflow/comments.py

```diff
@@ -0,0 +1,62 @@
+from typing import (
+    List,
+    Literal,
```

Comment:
You need to import `Literal` from typing_extensions because we still support older versions of Python :sweat_smile: 

```
from typing_extensions import Literal
```

Reply:
Thank you so much!